pkg/kudoctl/util/repo/repo.go

```diff
@@ -28,11 +28,29 @@ type Configuration struct {
 	Name string `json:"name"`
 }
 
+// Configurations is a collection of Configuration for Stringer
+type Configurations []*Configuration
+
 // Repositories represents the repositories.yaml file usually in the $KUDO_HOME
 type Repositories struct {
-	RepoVersion  string           `json:"repoVersion"`
-	Context      string           `json:"context"`
-	Repositories []*Configuration `json:"repositories"`
+	RepoVersion  string         `json:"repoVersion"`
+	Context      string         `json:"context"`
+	Repositories Configurations `json:"repositories"`
+}
+
+// String is a stringer function for Configuration
+func (c *Configuration) String() string {
+	return fmt.Sprintf("{ name:%v, url:%v }", c.Name, c.URL)
+}
+
+// String is a stringer function for Configurations
+func (c Configurations) String() string {
+	s := "repo configs: "
+	for _, config := range c {
```

Comment:
should we rather create an array of strings here and then do `strings.Join` for nicer output?